ext/hivemq-edge-openapi-2024.8-proposal.yaml

```diff
@@ -0,0 +1,5677 @@
+                $ref: '#/components/schemas/MetricList'
+          description: Success
+      summary: Obtain a list of available metrics
+      tags:
+        - Metrics
+        - Metrics Endpoint
+  /api/v1/metrics/{metricName}/latest:
+    get:
+      description: Obtain the latest sample for the metric requested.
+      operationId: getSample
+      parameters:
+        - description: The metric to search for.
+          in: path
+          name: metricName
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              examples:
+                metric-sample:
+                  description: Example response with metrics listed.
+                  summary: Metric Sample
+                  value:
+                    sampleTime: 2023-06-28T11:39:12.789+01
+                    value: 0
+              schema:
+                $ref: '#/components/schemas/DataPoint'
+          description: Success
+      summary: Obtain the latest sample for the metric requested
+      tags:
+        - Metrics
+        - Metrics Endpoint
+  /api/v1/management/domain/tags:
+    get:
+      description: Get the list of all domain tags created in this Edge instance
+      operationId: get-domain-tags
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get all domain tags
+      tags:
+        - Domain
+  /api/v1/management/domain/tags/schema:
+    get:
+      description: Get the data schema associated with the specified tags
+      operationId: get-tag-schemas
+      parameters:
+        - description: The list of tags to query the schema from
+          in: query
+          name: tags
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagSchemaList'
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get data schemas
+      tags:
+        - Domain
+  /api/v1/management/domain/topics:
+    get:
+      description: Get the list of all topic tags created in this Edge instance
+      operationId: get-domain-topics
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get all topic tags
+      tags:
+        - Domain
+  /api/v1/management/domain/topics/schema:
```

Comment:
what schema for what topic? Where does the schema come from? does it apply to all topics?